src/livekit/useLiveKit.ts

```diff
@@ -32,21 +42,35 @@ export function useLiveKit(): LiveKitState | undefined {
   // Create a React state to store the available devices and the selected device for each kind.
   const mediaDevices = useMediaDevicesState(room);
 
-  // Create local video track.
+  const [settingsDefaultDevices] = useDefaultDevices();
```

Comment:
A left over? We seem to call this function from the `useMediaDevicesState()`.

Reply:
How could i get the settings from the `useMediaDevicesState()`
I am calling it again to get the settings val in theis scope. since it is a hook that should be no perf disadvantag?